src/context/viewer-context.tsx

```diff
@@ -49,6 +67,7 @@ const defaultConfigOptions = {
   canvasHeight: "61.8vh",
   ignoreCaptionLabels: [],
   informationPanel: {
+    autoScroll: false,
```

Comment:
Following the new config shape, this should be adjusted to something like:

Note that I set enabled to `true`, I think that it'd be nice for that to be default.

```suggestion
    vtt: {
      autoScroll: {
        enabled: true,
        settings: defaultAutoScrollSettings,
      },
    },
```